Clean-up/clean_lambda.py

```diff
@@ -0,0 +1,55 @@
+"A script to remove unsubscribed products from the database"
+
+from os import environ as ENV
+import logging
+import json
+
+from dotenv import load_dotenv
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection, cursor
+
+
+def get_connection() -> connection:
+    "Establishes a connection with the database"
+    return connect(
+        user=ENV["DB_USER"],
+        password=ENV["DB_PASSWORD"],
+        host=ENV["DB_HOST"],
+        port=ENV["DB_PORT"],
+        database=ENV["DB_NAME"]
+    )
+
+
+def get_cursor(conn: connection) -> cursor:
+    "Returns a cursor for the database"
+    return conn.cursor(cursor_factory=RealDictCursor)
```

Comment:
Could test whether conn is a connection object:
```suggestion
def get_cursor(conn: connection) -> cursor:
    "Returns a cursor for the database"
    if not isinstance(connection, connection):
        raise TypeError('A cursor can only be constructed from a Psycopg2 connection object')
    return conn.cursor(cursor_factory=RealDictCursor)
```

Reply:
I've added

Reply:
Actually it messes up test

Reply:
works now